crates/bevy_sprite/src/collide_aabb.rs

```diff
@@ -55,3 +60,132 @@ pub fn collide(a_pos: Vec3, a_size: Vec2, b_pos: Vec3, b_size: Vec2) -> Option<C
         None
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    fn collide_two_rectangles(
+        // (x, y, size x, size y)
+        a: (f32, f32, f32, f32),
+        b: (f32, f32, f32, f32),
+    ) -> Option<Collision> {
+        collide(
+            Vec3::new(a.0, a.1, 0.),
+            Vec2::new(a.2, a.3),
+            Vec3::new(b.0, b.1, 0.),
+            Vec2::new(b.2, b.3),
+        )
+    }
+
+    #[test]
+    fn inside_collision() {
+        // Identical
+        #[rustfmt::skip]
+        let res = collide_two_rectangles(
+            (1., 1., 1., 1.),
+            (1., 1., 1., 1.),
+        ).expect("Collision expected");
+        assert_eq!(res, Collision::Inside);
```

Comment:
```suggestion
        );
        assert_eq!(res, Some(Collision::Inside));
```

Nit: There is a few places were `expect` could simply be replaced with adding `Some` to the `assert_eq`. It's not a big deal, I'm not sure everyone would agree it's an improvement, but here it is anyway.

Reply:
Thanks for the review! I like the suggestion.